Lib/http/server.py

```diff
@@ -491,6 +494,8 @@ def send_error(self, code, message=None, explain=None):
             body = content.encode('UTF-8', 'replace')
             self.send_header("Content-Type", self.error_content_type)
             self.send_header('Content-Length', str(len(body)))
+        if code == HTTPStatus.REQUESTED_RANGE_NOT_SATISFIABLE and range_size:
```

Comment:
- You can put this one in the above if.
- In addition, I would check for `range_size is not None` since `range_size = 0` is invalid and thus can be sent back (manually)